metagraph/src/main.cpp

```diff
@@ -86,6 +88,12 @@ int main(int argc, char *argv[]) {
         case Config::ALIGN:
             return cli::align_to_graph(config.get());
 
+        case Config::TRANSFORM_ANNO_TAX:
+            return cli::transform_anno_tax(config.get());
+
+        case Config::TAX_CLASSIFY:
```

Comment:
tax or taxo?

Pick one and use it consistently. (`#include "cli/taxo_classify.hpp"`, `Config::TAX_CLASSIFY`).

Reply:
I have changed everywhere to be "tax" only. Thanks!